lib/brain.js

```diff
@@ -1757,7 +1758,52 @@ Brain.prototype._start = function _start () {
     return this.smsCompliance()
   }
 
-  this._startAddressScan()
+  if (deviceConfig.complianceFeatures.printCashInWallet &&
+      deviceConfig.kioskPrinter &&
+      this.tx.cryptoCode === 'BTC') {
+
+    this._transitionState('cashInWaiting')
+
+    kioskPrinter.checkStatus(deviceConfig.kioskPrinter, (stats, err) => {
+      if (err) {
+        // TODO: New screen to surface this error to the UI
+        return
+      }
+
+      this.trader.createCashInWallet(this.tx, (res, err) => {
+
+        if (err) {
+          // TODO: New screen to surface this error to the UI
+          return
+        }
+
+        const pubAddr = res.body.toAddress
+        const privAddr = res.body.toAddress
+        const printerCfg = deviceConfig.kioskPrinter
+
+        kioskPrinter.printWallet(pubAddr, privAddr, printerCfg, (err) => {
+          if (err) {
+            // TODO: New screen to surface this error to the UI
+            return
+          }
+          this._handleScan(pubAddr)
+        })
+      })
+    })
+
+  }
+  else if (deviceConfig.complianceFeatures.printCashInWallet) {
```

Comment:
Shouldn't need ``else``, just return after previous ``if`` and start a new ``if``.